### PR TITLE
Include stub files 

### PR DESCRIPTION
**Pull request by Phxntxm**
Addresses #109 

Just creating this PR for now, if there are any comments let me know! Thanks for the help along with this. There are only a couple of things:

- As discussed in the issue, the scorer callable is a pain to try to type hint because of the limitations with ParamSpec (I really hope this is built out in the future...). I cannot for the life of my figure out a better way to type it than simply `Callable[...` either. 
- In string_metric I noticed that processor came after weights, and in one of the overloads is required. This isn't really a "typing" issue but that does seem strange as it makes it impossible to provide `levenshtein(s1, s2, processor)`. If you want the processor without the weights it *has* to come in keyword form, which feels to me like they should be keyword only arguments `levenshtein(s1, s2, *, weights = (1,1,1), processor, score_cutoff=0)`. But that's obviously a breaking change, and really not related to this PR... just a thing I noticed

### CODE REVIEW

**Comment by Phxntxm** [comment]
Oh and I can rebase to squash if that's desired, I just wanted to get this submitted before I headed out for a bit

**Comment by maxbachmann** [comment]
> As discussed in the issue, the scorer callable is a pain to try to type hint because of the limitations with ParamSpec (I really hope this is built out in the future...). I cannot for the life of my figure out a better way to type it than simply Callable[... either.

Yes I think, that this is the best we can do right now. At least we can validate the return type and use it for the function return type.

>  which feels to me like they should be keyword only arguments levenshtein(s1, s2, *, weights = (1,1,1), processor, score_cutoff=0). But that's obviously a breaking change, and really not related to this PR... just a thing I noticed

Yes these arguments should always be provided as keyword arguments. So having them keyword_only would make perfect sense. The only reason I did not use them back then was Python 2 compatibility (which was dropped since then). So I guess I will make them keyword only in the next major release, where this interface break is fine (so far I did not come across a single code base, which did not provide these parameters as keyword argument).

File: setup.py [hunk 1]
```diff
@@ -90,5 +90,6 @@ def build_extensions(self):
 if __name__ == "__main__":
     setup(
         cmdclass={'build_ext': BuildExt},
-        ext_modules = ext_modules
+        ext_modules = ext_modules,
+        package_data = {"rapidfuzz": ["*.pyi", "py.typed"]}
```
Comment (maxbachmann) [inline setup.py]: In other cases I list these files in the MANIFEST.in file.

**Comment by maxbachmann** [comment]
Other than that this looks good to me. As far as I could see this should be compatible with Python3.6 is this correct?

**Comment by Phxntxm** [comment]
Yes that's correct, all the main typing things came in version 3.5. The only newer one that would be useful in this typing set (as far as I can think) would be `typing.Final` for `__author__` and `__version__` and `__license__` in the init. That came in 3.8 though so I didn't use it

**Comment by maxbachmann** [comment]
Could you squash + rebase? Also I think the package_data in the setup.py is no longer needed, now that it is in the MANIFEST.in file.

**Comment by Phxntxm** [comment]
Done!

**Comment by maxbachmann** [comment]
Thanks

**Pull request merged**